CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ### Changed
 - In the diagnoses page genes associated with a disease are displayed using hgnc symbol instead of hgnc id
 ### Fixed
+- Refactored code in cases blueprints and variant_events adapter to use "disease" instead of "omim" to encompass also ORPHA terms
```

Comment:
```suggestion
- Refactored code in cases blueprints and variant_events adapter (set diseases for partial causative variants) to use "disease" instead of "omim" to encompass also ORPHA terms
```

Reply:
Updated accordingly